src/components/mdTable/mdTablePagination.vue

```diff
@@ -32,7 +32,7 @@
       },
       mdTotal: {
         type: [Number, String],
-        default: 'Many'
```

Comment:
The `Many` word is the default text for Google Projects, like on Gmail pagination on the top of the inbox.